pkg/scheduler/core/generic_scheduler.go

```diff
@@ -141,7 +141,7 @@ type genericScheduler struct {
 	extenders                []algorithm.SchedulerExtender
 	lastNodeIndex            uint64
 	alwaysCheckAllPredicates bool
-	cachedNodeInfoMap        map[string]*schedulernodeinfo.NodeInfo
+	nodeInfoSnapshot         schedulerinternalcache.NodeInfoSnapshot
```

Comment:
not really within scope of this PR, but do we really need these long import aliases for our own internal packages?

Reply:
Probably not, but as you noted, I don't want to include unrelated cleanups in this PR.